modules/registry-configuring-registry-storage-rhodf.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring_registry_storage/configuring-registry-storage-vsphere.adoc
+//
+// * registry/configuring_registry_storage/Configuring the registry for OpenShift Data Foundation.adoc
+
+:_content-type: PROCEDURE
+[id="registry-configuring-registry-storage-rhodf_{context}"]
+= Configuring the Image Registry Operator to using Red Hat OpenShift Data Foundation
+
+Configure the internal Image Registry to use {rh-storage-first} with custom certificate authority for S3 storage.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the {product-title} web console.
+* You installed the `oc` CLI.
+* Persistent storage provisioned for your cluster, such as Red Hat OpenShift Data Foundation.
+
+.Procedure
+
+In the {product-title} web console:
+
+* Navigate to the *Operator Hub* page and search for {rh-storage-first}.
+
+* Install {rh-storage} with all the default parameters.
+
+** Install the `storagesystem` with all the default parameters.
+
+* When {rh-storage} is ready, create `Objectbucketclaim.`
+
+** In the left navigation bar, click *Storage* → *Object Bucket Claims*.
+
+** Click *Create Object Bucket Claim*, and choose *ceph rgw sc*.
```

Comment:
IMO, could we assume customer has installed ODF operator, then use the ODF docs?  The odf installation is more complicated, it's different on vsphere and baremetal cluster, even on the cloud platform.
Those steps are not enough to describe clearly

Reply:
Thanks! I was concerned about this too. We can link to the ODF documentation and avoid replicating the steps.